website/content/api-docs/index.mdx

```diff
@@ -47,6 +47,11 @@ Responses from auth login methods that generate an authentication token are
 sent back to the client via JSON. The resulting token should be saved on the
 client or passed via the `X-Vault-Token` or `Authorization` header for future requests.
 
+## Parameter restrictions
```

Comment:
My suggestion: 
Several Vault APIs require specifying path parameters. The path parameter cannot end in periods. Otherwise, Vault will return a 404 unsupported path error.

Reply:
e057c8e